_docs/2017-01-10-technical.md

```diff
@@ -63,7 +63,7 @@ block (if you're the selected stakeholder), repeat.
 Listeners handle incoming messages and respond to them. 
 Various supplemental listeners will not be covered, focusing on the main ones instead.
 
-Mostly listeners use [Relay framework](/protocols/csl-application-level/#invreqdata-and-messagepart), which inludes three type of messages:
+Mostly listeners use [Relay framework](/technical/protocols/csl-application-level/#invreqdata-and-messagepart), which inludes three type of messages:
```

Comment:
"Mostly listeners" sounds strange. I know you didn't write it, but can you alter this sentence a bit?
"Listeners mostly use" would be better, the adverb before the subject and with no comma makes it hard to understand.